endpoints/globus/endpoint.go

```diff
@@ -91,6 +91,7 @@ func NewEndpoint(endpointName string) (core.Endpoint, error) {
 // authenticates with Globus using a client ID and secret to obtain an access
 // token (https://docs.globus.org/api/auth/reference/#client_credentials_grant)
 func (ep *Endpoint) authenticate(clientId uuid.UUID, clientSecret string) error {
+	fmt.Printf("%s\n%s\n", clientId.String(), clientSecret)
```

Comment:
this looks like it might be debugging code... or is it deliberate?

Reply:
You got me! Will delete.